lib/commons/text/is-icon-ligature.js

```diff
@@ -143,6 +139,14 @@ export default function isIconLigature(
   const firstChar = nodeValue.charAt(0);
   let width = canvasContext.measureText(firstChar).width;
 
+  // if the font uses zero-width characters that are not unicode characters
+  // it has to be an icon ligature (e.g. Affirm)
+  // @see https://github.com/dequelabs/axe-core/issues/3918
+  if (width === 0) {
+    font.numLigatures++;
+    return true;
```

Comment:
Why do you think a zero-width char is a ligature font? Unicode has at least one character that is supposed to have zero width (the zero-width space). There might be others, I'm not particularly well versed on unicode. This seems like a false assumption. Would it be better to loop over characters until you found one that has space?

Reply:
https://github.com/dequelabs/axe-core/pull/4102 will solve the zero-width unicode problem